style.css

```diff
@@ -0,0 +1,66 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+header {
+
+}
+body {
+  background-color: rgb(245, 245, 245);
+  margin: 0;
+}
+
+img {
+  max-width: 33%;
+  margin: 0;
+  padding: 12px;
+  border: hidden;
+  background-color: rgb(102, 102, 102);
+  align-items: center;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Since this applies to everything, I would suggest having this at the top of your CSS-file. Try to structure your CSS in the same way the HTML is ordered. So, start with `*`, then the `body`, then `h1`, etc as they come